### PR TITLE
fix: remove creation and uploading of release assets

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
The API server is meant to be consumed as part of
the UI. If it needs to be run standalone we provide
a container image. Don't see much benefit in providing
prebuilt binaries for now.

Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

**Review by ashokponkumar — APPROVED** [review]

File: .github/workflows/prbuild.yml [hunk 1]
```diff
@@ -2,23 +2,18 @@ name: PR Build
...
 on:
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
-
   build:
-    name: Build and Test
+    name: Build and test
```
Comment (ashokponkumar) [inline .github/workflows/prbuild.yml:9]: Aren't we naming it as "Build and Test" in other repos?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -2,66 +2,61 @@ name: Build
...
   push:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
-
-  build:
-    name: Build and Test
+  build_and_test:
+    name: Build and test
```
Comment (ashokponkumar) [inline .github/workflows/build.yml:9]: Aren't we naming it as "Build and Test" in other repos?

Comment (ashokponkumar) [reply .github/workflows/build.yml:9]: https://github.com/konveyor/move2kube/blob/17f03ff1498fb16503fd7b8411410d0b6afc1e90/.github/workflows/prbuild.yml#L10

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: .github/workflows/prbuild.yml (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply .github/workflows/prbuild.yml:9]: I changed it to this format in all the repos.

File: .github/workflows/build.yml (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply .github/workflows/build.yml:9]: Yeah I was working backwards from `move2kube-tests` move2kube-operator` `move2kube-ui` `move2kube-api`
Will get to `move2kube`

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**